### ISSUE TITLE
typedoc-plugin-pages : support of plantuml document

### ISSUE THREAD

**Issue by yboujraf** [mipatterson/typedoc-plugin-pages#11]
Dear @mipatterson 

I am following your plugin and it seems answering my request.

### PLANTUML RENDERING

I have external document in markdown where we have plantuml sequence diagram to document our project.

Do you think you will suport it. We already using plantuml plugin but don't now if we could use multiple plugin at same time to render the doc ?

### OPTION FILE

Which is the option to enable link with the code ? From my side I can see only the path but not as a link?

Best Regards,
Youssef

**Comment by loopingz** [comment]
I started working on this one: http://tplant.loopingz.com/ as I would also like to integrate it with my docs.

It is available on github.com/loopingz/tplantviewer

**Comment by mipatterson** [comment]
My concern here is that this plugin will never (and should never be expected to) perform every function that other plugins provide. It's just not sustainable. That being said, I do recognize that UML support seems to be popular among the users of TypeDoc.

Rather than implement this feature inside this plugin, I'm thinking it might be better to provide some sort of mechanism for third-party code to preprocess the Markdown that is rendered to pages. Though I'm hesitant to build a plugin framework for a plugin of a framework 😆.

I'm also wondering if the work I'm doing to support HTML pages might not be another avenue for achieving this in the short-term. This will allow you to include HTML-based pages in the build.

Let me give this some more thought. Perhaps there is a way to re-raise the TypeDoc events so that other plugins (like PlantUML plugins) would preprocess as normal without even knowing about this plugin...

**Comment by loopingz** [comment]
Yes, I do not think this stands in this plugin. The uml plugin should be a separate plugin that trigger a tplant export and reprocess it as other pages. That is why I started playing with how to display it

**Comment by airtonix** [comment]
just let people describe markdown plugins that already perform this function

**Comment by loopingz** [comment]
@airtonix not sure what you meant

**Comment by airtonix** [comment]
@loopingz it was a casual drive by comment at @mipatterson.

typedoc-plugin-pages should just expose a configuration option to control which marked plugins to use.
